tests/integration_tests/functional/test_feat_parity.py

```diff
@@ -297,27 +272,18 @@ def test_feat_parity_cpuid_sec(
         raise Exception("Unsupported CPU vendor.")
 
     check_cpuid_feat_flags(
-        vm_builder,
-        inst_set_cpu_template,
-        microvm,
-        guest_kernel,
-        disk,
+        vm,
         must_be_set,
         must_be_unset,
     )
 
 
-def test_feat_parity_msr_arch_cap(
-    vm_builder, inst_set_cpu_template, microvm, guest_kernel, disk
-):
+def test_feat_parity_msr_arch_cap(vm, inst_set_cpu_template):
```

Comment:
It isn't a problem now, but `vm` is constructed with `inst_set_cpu_template` implicitly, so one changing the 2nd argument will have to remember to change the 1st as well. Is there a way to make it less error prone (eg query the template from the `vm` itself?

Reply:
We should be able to query it from the API. Let me try that...